Makefile

```diff
@@ -0,0 +1,40 @@
+.PHONY: dependencies clean lint fmt syntax-check
+
+all: dependencies
+
+# clean removes and re-initializes the directories that have their contents
+# ignored by git. This will force a re-installation of the requirements, roles,
+# etc.
+clean:
+	rm -rf my-logs
```

Comment:
If I want to run linting locally, I don't want to delete all logs everytime. To force re-installation of the requirements & co. maybe just deleting `my-logs/requirements-installed.txt` file should be enough.

Reply:
I would recommend a separate command to just remove the requirements file. 
`clean` should clean everything from the repo, as if it was recently cloned. 

either way, recommend this as a follow up if we want to discuss changing this more.